src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = (num1) => {
+    return function(num2) {
```

Comment:
```suggestion
    return (num2) => {
```
You can use here arrow function too

Reply:
agree